test/e2e/testnet/testnet.go

```diff
@@ -340,7 +326,11 @@ func (t *Testnet) Start() error {
 		for i := 0; i < 10; i++ {
 			resp, err := client.Status(context.Background())
 			if err != nil {
-				return fmt.Errorf("node %s status response: %w", node.Name, err)
+				if i == 9 {
+					return fmt.Errorf("node %s status response: %w", node.Name, err)
+				}
+				time.Sleep(time.Second)
```

Comment:
[nit][optional]
is there a chance the context gets cancelled somewhere else? If so, it would be good to have a ticker here along with a `case <-ctx.Done()` in case the context was done somewhere else and this stays pending for 9sec before it stops.

Reply:
The context is created in the scope of the function with `context.Background()`.